src/execution/parsing/dag_parser.c

```diff
@@ -48,15 +50,13 @@ static int _ParseDAGLoadArgs(RedisModuleCtx *ctx, RedisModuleString **argv, int
     for (size_t argpos = 2; argpos < argc && number_loaded_keys < n_keys; argpos++) {
         RedisModuleString *key_name = argv[argpos];
         const char *arg_string = RedisModule_StringPtrLen(key_name, &arg_len);
-        if (!strcasecmp(arg_string, chaining_operator))
-            break;
         RAI_Tensor *t;
         RedisModuleKey *key;
         const int status =
             RAI_GetTensorFromKeyspace(ctx, key_name, &key, &t, REDISMODULE_READ, err);
         if (status == REDISMODULE_ERR) {
-            RedisModule_Log(ctx, "warning",
-                            "on DAGRUN's LOAD could not load tensor %s from keyspace", arg_string);
+            RedisModule_Log(ctx, "warning", "Could not LOAD tensor %s from keyspace into DAG",
```

Comment:
The previous phrasing is more informative

Reply:
Right, but now we use this parser from both AI.DAGRUN(_RO) and AI.DAGEXECUTE(_RO) commands, so I changed to something that will fit both scenarios.